versioned_docs/version-2.0/concepts/tracing/tracing.mdx

```diff
@@ -68,3 +68,15 @@ To learn more about how feedback is stored in the application, see [this referen
 `Metadata` is a collection of key-value pairs that can be attached to runs. Metadata can be used to store additional information about a run, such as the version of the application that generated the run, the environment in which the run was generated, or any other information that you want to associate with a run.
 Similar to tags, you can use metadata to filter runs in the LangSmith UI, and can be used to group runs together for analysis. [Learn how to add metadata to your traces](../how_to_guides/tracing/add_metadata_tags)
 ![Metadata](../static/metadata.png)
+
+## Data Storage and Retention
+
+As of Tuesday, May 21, 2024, LangSmith (SaaS) retains trace data for a maximum of 400 days past the date and time the trace was inserted into the LangSmith trace database.
```

Comment:
this may end up being Wednesday UTC!

Reply:
Hence draft.  😁   We'll change accordingly.